components/searchable_channel_list.jsx

```diff
@@ -193,7 +195,8 @@ export default class SearchableChannelList extends React.Component {
                             id='searchChannelsTextbox'
                             ref='filter'
                             className='form-control filter-textbox'
-                            placeholder={localizeMessage('filtered_channels_list.search', 'Search channels')}
+                            placeholder={{id: 'filtered_channels_list.search', defaultMessage: 'Search channels'}}
```

Comment:
Because we don't have any way to infer this from the `mmjstool i18nextract` tool, we should use here something like `{id: t('filtered_channels_list.search'), defaultMessage: 'Search channels'}`.